starknet_py/contract.py

```diff
@@ -484,10 +510,7 @@ def __init__(
                 cairo_version=cairo_version,
             )
         except ValidationError:
-            warnings.warn(
-                "Make sure valid ABI is used to create a Contract instance: "
-                "Cairo1 contract ABIs are currently unsupported."
-            )
+            warnings.warn("Make sure valid ABI is used to create a Contract instance")
```

Comment:
A warning here, but exception wrapping in `deploy`, is there a reason for the difference?

Reply:
Hmm, I think we can throw in both places